e2e/specs/pyplot.spec.ts

```diff
@@ -35,9 +35,6 @@ describe("st.pyplot", () => {
       which: 82 // "r"
     });
 
-    // Wait for 'stale-element' class to go away, so the snapshot looks right.
```

Comment:
Why remove this?

Reply:
may have been an accident. I'll restore it.